### PR TITLE
Fixing small move bugs

### PR DESCRIPTION
**Pull request by kenny2892**

### CODE REVIEW

File: Anature/src/application/anatures/moves/moves/JustDamageDealing.java [hunk 1]
```diff
@@ -8,6 +8,6 @@
 	@Override
 	public void activateMove(IAnature source, IAnature target)
 	{
-		target.takeDamage(calculateDamage(source, target, false));
+		target.takeDamage(calculateDamage(source, target, !isPhysicalAttack()));
```
Comment (HMRich) [inline Anature/src/application/anatures/moves/moves/JustDamageDealing.java:11]: So all moves that are non-pysical are considered Special attacks? I made a couple of special attacks that are physical but also special.

Just making sure that you understand that the flag there is whether the move is special or not.

**Review by HMRich — COMMENTED** [review]
Hey just had a comment/question.

File: Anature/src/application/anatures/moves/moves/JustDamageDealing.java (hunk 1 above)
Comment (kenny2892) [reply Anature/src/application/anatures/moves/moves/JustDamageDealing.java:11]: yeah its either special or physical. Physical means use atk and def while special means use the special stats.

Comment (HMRich) [reply Anature/src/application/anatures/moves/moves/JustDamageDealing.java:11]: Then why have `isPhysicalAttack` as the third argument for `calculateDamage` that flag is supposed to be identifying if the damage that will be calculated use attack and defense or special attacks and special defense.

**Review by HMRich — APPROVED** [review]
👍

**Pull request merged**